client/my-sites/stats/feedback/index.tsx

```diff
@@ -145,22 +162,31 @@ function StatsFeedbackController( { siteId }: FeedbackProps ) {
 
 	const handleButtonClick = ( action: string ) => {
 		switch ( action ) {
-			case FEEDBACK_ACTION_SEND_FEEDBACK:
+			case ACTION_SEND_FEEDBACK:
 				setIsFloatingPanelOpen( false );
 				setIsOpen( true );
 				break;
-			case FEEDBACK_ACTION_DISMISS_FLOATING_PANEL:
+			case ACTION_DISMISS_FLOATING_PANEL:
 				setIsFloatingPanelOpen( false );
 				updateFeedbackPanelHibernationDelay();
+
+				// stats_feedback_action_dismiss_floating_panel
+				trackStatsAnalyticsEvent( `stats_feedback_${ ACTION_DISMISS_FLOATING_PANEL }` );
 				break;
-			case FEEDBACK_ACTION_LEAVE_REVIEW:
+			case ACTION_LEAVE_REVIEW:
 				setIsFloatingPanelOpen( false );
 				window.open( FEEDBACK_LEAVE_REVIEW_URL );
 				break;
 			// Ignore other cases.
 		}
 	};
 
+	const onModalClose = () => {
+		setIsOpen( false );
+
+		trackStatsAnalyticsEvent( 'stats_feedback_action_close_form_modal' );
```

Comment:
Thanks for adding this! I think it's better to have this here but I'm also thinking it makes sense to have an event specific to dismissing the modal. This event will fire for both a successful submission and a cancellation. So while it represents the flow of the UI, it's not as useful for explicitly tracking users who back out of the flow.

What do you think?

Reply:
The follow-up PR is here: https://github.com/Automattic/wp-calypso/pull/94574.